python/python/glide/async_commands/server_modules/json.py

```diff
@@ -139,6 +139,55 @@ async def get(
     return cast(bytes, await client.custom_command(args))
 
 
+async def arrappend(
+    client: TGlideClient,
+    key: TEncodable,
+    values: List[TEncodable],
+    path: Optional[TEncodable] = None,
+) -> TJsonResponse[int]:
+    """Appends one or more `values` to the JSON array at the specified `path` within the JSON document stored at `key`.
+
+    See https://valkey.io/commands/json.append/ for more details.
```

Comment:
This link is not found. Do we use `https://redis.io/docs/latest/commands/json.arrappend/` instead?

Reply:
forgot to remove it, you can ignore